### DISCUSSION TITLE
Collection for Failed Transaction Cases in Coordinator Service

### DISCUSSION THREAD

**Discussion by mzxyz** [subquery/subql#1037] [Frontier TestNet - Bug Reporting]
**Hey our indexers:**

There has several reasons to make the transaction failed. The retry mechanism can make sure the failed transaction to be processed properly. However, we still want to collect the failed cases to investigate the root cause of the problem.

So feel free to add the screenshoot of you failed transaction log which is in `Network` -> `Service Log`.

**Note:**
Please also attach the version of coordinator service

**Comment by mzxyz** [comment]
Coordinator: `v0.1.10`

<img width="1189" alt="Screen Shot 2022-05-18 at 9 20 13 PM" src="https://user-images.githubusercontent.com/8177474/169005105-3b986b2d-582a-46bd-a00f-f6aca4e34b62.png">

**Comment by yasiryagi** [comment]
After upgrade to v1.0.0 this error came back, it was reported in issue [917](https://github.com/subquery/subql/discussions/917)


[subquery_error.txt](https://github.com/subquery/subql/files/8715123/subquery_error.txt)

**Comment by jim-counter** [comment]
![image](https://user-images.githubusercontent.com/11335994/169258645-4d44277e-259b-4395-83ec-1faddcacb50c.png)

**Comment by bobyD** [comment]
Coordinator: `v1.0.0`

<img width="1433" alt="Снимок экрана 2022-05-17 в 11 39 08" src="https://user-images.githubusercontent.com/69699206/169477494-cc666851-a168-46f7-b440-0a2f076678f8.png">
<img width="1400" alt="Снимок экрана 2022-05-17 в 15 22 24" src="https://user-images.githubusercontent.com/69699206/169477553-752416a9-0ee6-45ba-8840-495e09a6370b.png">

_QuantNode team_

**Comment by jim-counter** [comment]
![image](https://user-images.githubusercontent.com/11335994/169651024-29294206-58c9-436f-8379-f3e55ad7288e.png)
